SPEC.md

```diff
@@ -2566,6 +2580,26 @@ Array[Integer] env2 = [1, 2, 3]
 Array[String] env2_param = prefix("-f ", env2) # ["-f 1", "-f 2", "-f 3"]
 ```
 
+## X select_first(Array[X?])
+
+:pig2: [Cromwell supported](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
+
+Given an array of optional values, `select_all` will select the first defined value and return it. Note that this is a runtime check and you should be confident that a defined value will exist: if no defined value is found when select_first runs, the workflow will fail.
```

Comment:
"you should be confident that a defined value will exist" -> "requires that at least one defined value exists at the time the check is made" ?